tagging/routes/routeGetMostUsedTags.go

```diff
@@ -54,3 +55,37 @@ func routeGetMostUsedTags(r *http.Request, a routes.Arguments) (int, interface{}
 		"mostUsedTags": tagsList,
 	}
 }
+
+func routeGetMostUsedTagsHistory(r *http.Request, a routes.Arguments) (int, interface{}) {
+	logger := jsonlog.LoggerFromContextOrDefault(r.Context())
+	u := a[users.AuthenticatedUser].(users.User)
+	tx := a[db.Transaction].(*sql.Tx)
+	dbRes, err := models.MostUsedTagsHistoryByUser(tx, u.Id)
+	if err != nil {
+		logger.Error("Could not fetch most used tags.", err.Error())
+		return http.StatusInternalServerError, nil
+	}
+
+	if dbRes == nil {
+		return http.StatusInternalServerError, map[string]interface{}{
```

Comment:
Same here, you don't have to return a `map[string]interface{}`, you can just go and return something like `errors.New("No history reports available")` (unless the `map[string]interface{}` looks much better or something like that)